frontend/package.json

```diff
@@ -1,19 +1,21 @@
 {
-  "name": "umbra-frontend",
+  "name": "@umbra/frontend",
   "version": "0.0.1",
   "description": "Send and receive stealth payments with the Umbra protocol",
   "productName": "Umbra",
   "author": "Matt Solomon <matt@mattsolomon.dev>",
   "private": true,
   "scripts": {
     "lint": "eslint --ext .js,.ts,.vue ./",
-    "prettier": "npx prettier --write .",
+    "prettier": "prettier --write .",
     "test": "echo \"No test specified\" && exit 0",
-    "dev": "npx quasar dev",
-    "build": "npx quasar build"
+    "dev": "quasar dev",
+    "build": "quasar build",
+    "clean": "rimraf build"
   },
   "dependencies": {
     "@quasar/extras": "^1.0.0",
+    "@umbra/umbra-js": "^0.0.1",
```

Comment:
How does it know to get this package from the `umbra-js` folder instead of npm? Is that from `lerna.json`?

Reply:
[Yarn workspaces](https://classic.yarnpkg.com/en/docs/workspaces/)!